images/singleuser-sample/Dockerfile

```diff
@@ -58,4 +32,11 @@ RUN adduser --disabled-password \
 
 USER $NB_USER
 
+COPY requirements.txt /tmp/requirements.txt
```

Comment:
pip install still needs to be done as root, otherwise it does a `--user` install, which won't work for permission/path/volume mount reasons.